src/styles/card.scss

```diff
@@ -0,0 +1,110 @@
+.card {
+  width: 200px;
```

Comment:
can be moved to a variable so as not to hardcode, for example: $card-width: 200px;